src/cpp/number-9.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+#include <random>
+
+int main() {
+    srand(time(NULL));
+
+    if (rand() % 9 + 1 == 9) {
```

Comment:
Doing modulo 10 does the same thing
```suggestion
    if (rand() % 10 == 9) {
```